dietpi/dietpi-software

```diff
@@ -12376,6 +12402,63 @@ _EOF_
 
 		fi
 
+    software_id=181 # Paper
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+      
+      Create_User -d /opt/papermc papermc # This creates a system user named papermc with /opt/papermc as its home dir.
+      java -jar /opt/papermc/paperclip.jar nogui
+      echo eula=true > /opt/papermc/eula.txt
+      # Bedrock compatibility
+      G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+      if G_WHIP_YESNO "Would you like to install the Geyser and Floodgate\n
+      plugins for compatibility with Bedrock Edition?
+      \n Note that this may be buggy."; then
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar' /opt/papermc/plugins/Geyser-Spigot.jar
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar' /opt/papermc/plugins/floodgate-bukkit.jar
+      fi
+      # Service
+      cat << _EOF_ > /etc/systemd/system/papermc.service
+[Unit]
+Description=papermc (DietPi)
+
+[Service]
+User=papermc
+SyslogIdentifier=papermc
+WorkingDirectory=/opt/papermc
+ExecStart=$(command -v java) -jar /opt/papermc/paperclip.jar
+
+[Install]
+WantedBy=multi-user.target
+_EOF_
+      # Configuration generation
+      if [[ ! -f '/opt/papermc/permissions.yml' ]]
+      then
+        G_EXEC_DESC='Starting Paper to pre-create config file' G_EXEC systemctl start papermc
+        i=0
+        until [[ -f '/opt/papermc/permissions.yml' ]] || (( $i >= $timeout ))
+        do
+          ((i++))
+          G_DIETPI-NOTIFY -2 "Waiting for Paper config file to be created ($i/$timeout)"
+          sleep 1
+        done
+      fi
+      sleep 1
+      G_EXEC_NOHALT=1 G_EXEC systemctl stop papermc
+      sleep 1
+      if [[ ! -f '/opt/papermc/permissions.yml' ]]
+      then
+        G_DIETPI-NOTIFY 1 'Paper config file creation timed out'
+      else
+        if [[ -f '/opt/papermc/plugins/Geyser-Spigot.jar' ]] && [[ -f '/opt/papermc/plugins/floodgate-bukkit.jar' ]]
+        then
+        G_CONFIG_INJECT 'auth-type: online' 'auth-type: floodgate' /opt/papermc/plugins/Geyser-Spigot/config.yml
```

Comment:
Use it the following way:
```
G_CONFIG_INJECT 'auth-type:[[:blank:]]' 'auth-type: floodgate' /opt/papermc/plugins/Geyser-Spigot/config.yml
```
That way `floodgate` authentication is applied regardless which setting was present before.